getting-started/templates/systemlink-values.yaml

```diff
@@ -94,6 +94,12 @@ global:
     #  <ATTENTION> - If you do *not* have your own MongoDB instance set this (`global.mongodb.install`) to `true`.
     ##
     install: false
+  ##
+  ## Specify secrets containing trusted certificates that should be installed in all the necessary applications.
```

Comment:
Can we be clear in the comment that these are certificate authority public certificates which the user has hand-deployed as secrets on the cluster prior to installing SystemLink?

Reply:
Specified that the secrets must be deployed manually.